torchtext/datasets/iwslt2017.py

```diff
@@ -99,6 +98,17 @@
 DATASET_NAME = "IWSLT2017"
 
 
+def _filter_clean_cache(cache_decompressed_dp, full_filepath, uncleaned_filename):
```

Comment:
It would be nice to pull this into dataset_utils.py but I think maybe this can wait until torchdata is strictly a dependency so we don't need to do extra conditional imports. Thoughts, @parmeet @Nayef211?

Reply:
That sounds good to me. Good we put in a TODO comment here

Reply:
Done!